__tests__/organizations.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  GraphQLTestClient,
+  GraphQLTestingState,
+  initializeGraphQLTesting,
+  MockContext,
+  saveFixtures,
+  testQueryErrorCode,
+} from './helpers';
+import { Feed, Organization, User } from '../src/entity';
+
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../src/db';
+
+import { usersFixture } from './fixture/user';
+import { ContentPreferenceOrganization } from '../src/entity/contentPreference/ContentPreferenceOrganization';
+import { ContentPreferenceStatus } from '../src/entity/contentPreference/types';
+import { OrganizationMemberRoles } from '../src/roles';
+
+let con: DataSource;
+let state: GraphQLTestingState;
+let client: GraphQLTestClient;
+let loggedUser: string | null = null;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+  state = await initializeGraphQLTesting(
+    () => new MockContext(con, loggedUser, []),
+  );
+  client = state.client;
+});
+
+beforeEach(async () => {
+  loggedUser = null;
+  jest.clearAllMocks();
+});
+
+describe('organizations', () => {
```

Comment:
Maybe we can let go of the `describe` that wraps everything? Just a personal preference though, non-blocking 🙏

Reply:
Yeah probably